### PR TITLE
🌐 Add Russian translation for `docs/ru/docs/deployment/docker.md`

### PR DESCRIPTION
**Pull request by Xewus**
This is a really voluminous page...  
:triumph:

[Original  page](https://fastapi.tiangolo.com/deployment/docker/)
[Translated page](https://84ca810e.fastapitiangolo.pages.dev/ru/deployment/docker/)

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4b4940b48869af6ec9622cdf82e3c3fc00668c6d at: https://64c7348ed37abc6ba1131365--fastapi.netlify.app

**Comment by tiangolo** [comment]
📝 Docs preview for commit d1253a650a5e1cd7f9898ce736f74371aea9defe at: https://64c7426e71ccc66c38699754--fastapi.netlify.app

**Review by Yois4101 — APPROVED** [review]

**Review by Artem4es — APPROVED** [review]

**Review by Artem4es — APPROVED** [review]

File: docs/ru/docs/deployment/docker.md [hunk 1]
```diff
@@ -0,0 +1,700 @@
...
+
+    Опция `--upgrade` указывает `pip` обновить библиотеки, емли они уже установлены.
+
+    Ка и в предыдущем шаге с копированием файла, этот шаг также будет использовать **кэш Docker** в случае отсутствия изменений.
+
+    Использрвание кэша, особенно на этом шаге, позволит Вам **сэкономить** кучу времени при повторной сборке образа, так как зависимости будут сохранены в кеше, а не **загружаться и устанавливаться каждый раз**.
+
+5. Скопируйте директорию `./app` внутрь директории `/code` (в контейнере).
+
+    Так как в этой директории расположен код, который **часто изменяется**, то использование **кэша** на этом шаге будет наименее эффективно, а значит лучше поместить этот шаг **ближе к концу** `Dockerfile`, дабы не терить выгоду от оптимзации предыдущих шагов.
```
Comment (Yois4101) [inline docs/ru/docs/deployment/docker.md:215]:
```suggestion
    Так как в этой директории расположен код, который **часто изменяется**, то использование **кэша** на этом шаге будет наименее эффективно, а значит лучше поместить этот шаг **ближе к концу** `Dockerfile`, дабы не терять выгоду от оптимизации предыдущих шагов.
```

File: docs/ru/docs/deployment/docker.md [hunk 2]
```diff
@@ -0,0 +1,700 @@
...
+Есть два основных подхода, которые Вы можете использовать при запуске контейнеров (Docker, Kubernetes и т.п.).
+
+### Множество контейнеров
+
+Когда Вы запускаете **множество контейнеров**, в каждом из которых работает **только один процесс** (например, в кластере **Kubernetes**), может возникнуть необходимость иметь **отдельный контейнер**, который осуществит **предварительные шаги перед запуском** остальных контейнеров (например, применяет миграции к базе данных).
+
+!!! info "Информация"
+    При использовании Kubernetes, это может быть <a href="https://kubernetes.io/docs/concepts/workloads/pods/init-containers/" class="external-link" target="_blank">Инициализирующий контейнер</a>.
+
+При отсутствии такой необходимости (допустим, не нужно применять миграции к базе данных, а только проверяете, что она готова принимать соединения), Вы можете проводить такую проверку в каждом контейнере перед запуском его основного процесса и запускать все контейнеры **одновременно**.
```
Comment (Yois4101) [inline docs/ru/docs/deployment/docker.md:510]:
```suggestion
При отсутствии такой необходимости (допустим, не нужно применять миграции к базе данных, а только проверить, что она готова принимать соединения), Вы можете проводить такую проверку в каждом контейнере перед запуском его основного процесса и запускать все контейнеры **одновременно**.
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit 2d46a66abf9d6a8291be251b67bd94e2ebd7bd47 at: https://4c4bf20b.fastapitiangolo.pages.dev

File: docs/ru/docs/deployment/docker.md [hunk 3]
```diff
@@ -0,0 +1,700 @@
+# FastAPI в контейнерах - Docker
+
+При развёртывании приложений FastAPI часто начинают с создания **образа контейнера на основе Linux**. Обычно для этого используют <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a>. Затем можно развернуть такой контейнер на сервере одним из нескольких способов.
```
Comment (Xewus) [inline docs/ru/docs/deployment/docker.md:3]:
```suggestion
При развёртывании приложений FastAPI, часто начинают с создания **образа контейнера на основе Linux**. Обычно для этого используют <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a>. Затем можно развернуть такой контейнер на сервере одним из нескольких способов.
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit 922439fb70e8e86b662649011cdd1026853da6db at: https://c510282a.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 215d4589d216fb64ced7f5026d5d111f9e883fd0 at: https://84ca810e.fastapitiangolo.pages.dev

**Review by Yois4101 — APPROVED** [review]

**Pull request merged**